### ISSUE TITLE
🐛 Bug Report: Traces showing up as dependencies in Azure Application Insights

### ISSUE THREAD

**Issue by joshuaphelpsms** [traceloop/openllmetry#408]
### Which component is this bug for?

Traceloop SDK

### 📜 Description

I'm not sure if this bug report should be in the `docs` repo or here, but I am trying to send traces to Azure Application Insights using traceloop and getting different results that what is documented.

### 👟 Reproduction steps

Run the sample code from https://www.traceloop.com/docs/openllmetry/integrations/azure

### 👍 Expected behavior

I expected to see a trace but instead am seeing the call show up as a dependency in Application Insights

The screenshot from the [docs](https://www.traceloop.com/docs/openllmetry/integrations/azure) shows there are 4 traces and there seems to be an extra `llm_span` that I don't have on my end.

### 👎 Actual Behavior with Screenshots

When I ran the code form the docs, `Traces` are coming through as `Dependencies`. From the screenshot below, you can see that `Traces` are 0. 
![Screenshot 2024-02-06 at 10 54 24 AM](https://github.com/traceloop/openllmetry/assets/88395136/ecaa6189-78b1-421e-b91c-cd4c4c37e8fc)

### 🤖 Python Version

3.11.5

### 📃 Provide any additional context for the Bug.

In addition, the code from the documentation needs some updates:
1. Missing `os` import
2. `response['choices'][0]['message']['content']` should be updated to `response.choices[0].message.content`.

> note I would normally just submit a PR with these changes but since I am running into different behavior than what is documented, I chose to open an issue instead.

### 👀 Have you spent some time to check if this bug has been raised before?

- [X] I checked and didn't find similar issue

### Are you willing to submit PR?

Yes I am willing to submit a PR!

**Comment by joshuaphelpsms** [comment]
I am unsure if it is bug on the Azure side. The sample image shows `4 Traces` near the bottom of the screen and so I expected to have the same experience. When I ran the sample, I see `0 Traces`. It also shows the `Telemetry type` as `dependency`. I am wondering if there is something I am missing and how I can get this to show up as a trace? If it is problem on the Azure side then I can move that conversation over there. 

There are also some misalignments with the code and doc screenshot so thought I should bring that up as well.

**Comment by joshuaphelpsms** [comment]
I was able to look into this more and this does not seem to be a problem with OpenLLMetry. The screenshot shows traces because the code that was used to generate it was doing some additional calls that are not in the current sample code. 

I will submit a PR on the docs repo addressing:
1. missing import
2. Fixing `response['choices'][0]['message']['content']` -> `response.choices[0].message.content` for OpenAI v1
3. Updated screenshot

Thanks!
